include/open62541/server.h

```diff
@@ -1553,6 +1554,38 @@ UA_Server_updateCertificate(UA_Server *server,
                             UA_Boolean closeSessions,
                             UA_Boolean closeSecureChannels);
 
+#ifdef UA_ENABLE_FILETYPE_OBJECT_SUPPORT
+
+/* Add the FileType Object instance to the server information model
+ * and register method callbacks
+ *
+ * @param server The server object
+ * @param requestedNewNodeId The requested NodeId of the fileType Object instance
+ * @param filePath Path to the file
+ * @return UA_STATUSCODE_GOOD on success
+ */
+UA_StatusCode UA_EXPORT
+UA_Server_addFile(UA_Server *server, const UA_NodeId requestedNewNodeId,
```

Comment:
For consistency with `registerFileNode`, rename this to `UA_Server_addFileNode`.